pasqal_cloud/__init__.py

```diff
@@ -54,6 +55,40 @@
 from pasqal_cloud.utils.responses import JobCancellationResponse, PaginatedResponse
 from pasqal_cloud.workload import Workload
 
+from ._version import __version__, deprecation_date
+
+DEPRECATION_WARNING_PERIOD = timedelta(days=30)
+
+# Ensure DeprecationWarnings are not filtered out at runtime
+simplefilter("always", DeprecationWarning)
```

Comment:
I'm not familiar with this, what does this do and what is the idea behind it?

Reply:
This is to ensure python doesn't filter out Deprecation warnings at runtime. I was encountering them fine in the tests but at runtime, locally the warnings didn't show up. This fixed it.